src/components/Login.js

```diff
@@ -66,8 +66,8 @@ const Login = (props) => {
 
   const history = useHistory();
   const location = useLocation();
-  const { from } = location.state || { from: { pathname: '/' } };
```

Comment:
I can't test this at the moment, but it looks like this will cause us to _always_ forget the path. Instead, we should only forget it when a user _explicitly_ logs out via the Account page.

If they are redirected to the login page (e.g., because their session expired or they weren't logged in to begin with), the path they entered should be remembered.